packages/data/src/components/use-select/index.js

```diff
@@ -148,12 +148,11 @@ export default function useSelect( _mapSelect, deps ) {
 				errorMessage += `\nThe error may be correlated with this previous error:\n`;
 				errorMessage += `${ latestMapOutputError.current.stack }\n\n`;
 				errorMessage += 'Original stack trace:';
-
-				throw new Error( errorMessage );
```

Comment:
Does this mean we're basically silencing all errors from useSelect?

Reply:
Errors will be logged, not thrown no.